app/grandchallenge/algorithms/views.py

```diff
@@ -335,6 +338,82 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
+class AlgorithmImageMarkDesired(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, View
+):
+    permission_required = "algorithms.change_algorithm"
+    raise_exception = True
+
+    @cached_property
+    def algorithm(self) -> Algorithm:
```

Comment:
FYI, I know that they're there already in some places but we shouldn't add type annotations as they're not being checked, it's worse to have them as they could drift and be wrong.